src/views/UrbitOS/NetworkKeys.tsx

```diff
@@ -280,8 +280,41 @@ export default function UrbitOSNetworkKeys({
     ? 'Set Network Keys'
     : 'Initiate Network Keys';
 
-  const usageMessage =
-    'With your new network keys, you also have a new keyfile. Download your new keyfile to boot your OS.';
+  const bootMessage = () => (
+    <Text as="p" textAlign={'center'}>
+      Download your new Network Key to boot your ship. Be sure to delete your
+      old pier before booting with this new key.
+    </Text>
+  );
+
+  const resetMessage = () => (
+    <>
+      <Text as="p" textAlign={'center'}>
+        Copy the contents of the Network Key file and run
+      </Text>
+      <Text
+        as="code"
+        mono
+        bg={'washedGray'}
+        padding={2}
+        borderRadius={2}
+        textAlign={'center'}>
+        |rekey 'keyfile_contents'
+      </Text>
+      <Text as="p" textAlign={'center'}>
+        using the dojo on your running ship.
+      </Text>
+    </>
+  );
+
+  const waitMessage = () => (
+    <Text as="p" textAlign={'center'}>
+      <strong>Important:</strong> you will need to wait until the next L2 batch
+      is submitted to the roller to boot your ship. If you do not wait, then you
+      will run into a key mismatch error and will not be able to communicate
+      with other ships on the network.
```

Comment:
I think it will just prevent you from booting. We have preboot validation in place that checks the provided keys against the roller.